cilium-dbg/cmd/bpf_ct_flush.go

```
@@ -31,6 +32,20 @@ func init() {
 
 func flushCt() {
 	ipv4, ipv6 := getIpEnableStatuses()
+
+	nat4, nat6 := nat.GlobalMaps(nil, ipv4, ipv6)
+	if err := nat4.Open(); err != nil {
```

additional nil check necessary before opening the map (same for ipv6) - otherwise it panics

---

previously it was initialized with `ctmap.InitMapInfo(nil, true, true, true)` (IPv4&IPv6 always "enabled") - but i think it's better how you implemented it with fetching the active IP families first and only flush these maps.

But i'd prefer if we could fail the command if it's not able to open the map (instead of nil-ing it and continue)